### DISCUSSION TITLE
Jest In-memory database + NestJS

### DISCUSSION THREAD

**Discussion by navyaharish** [jmcdo29/testing-nestjs#1390] [General]
Hi there, is there any repo with use cases for testing a nestJs application + Mongo using Jest's in-memory db?

**Comment by jmcdo29** [comment]
I don't have anything in this repo for using an in-memory db for mongo, but it is a use case for those who want to pursue that avenue. At that point, you're writing integration tests, rather than unit tests, but good test coverage can come from it instead
